l10n_ch_pain_direct_debit/models/account_payment_order.py

```diff
@@ -413,12 +418,13 @@ def generate_xml_ch_dd_file(self):
                     "self.name + '-' "
                     "+ str(requested_date) + '-' + priority + "
                     "'-' + local_instrument",
-                    prio, local_inst, False, False, req_date, {
+                    # TODO replace the False just before eval_ctx by sepa boolean
```

Comment:
What's this? We don't want to leave todos in OCA repos.

Reply:
Ah yes I forgot to remove that